pyproject.toml

```diff
@@ -12,8 +12,8 @@ readme = "README.md"
 license = {text = "BSD 3-Clause License"}
 dynamic = ["version"]
 dependencies = [
-    "numpy >=1.23", "cvxpy >=1.2", "scikit-learn >=1.2",
-    "scipy >=1.9", "joblib", "cvxopt"
+    "numpy >=1.23", "cvxpy >=1.3.0", "scikit-learn >=1.2.1",
+    "scipy >=1.9", "joblib", "cvxopt >= 1.3.0", "scs>=3.2.2"
```

Comment:
SCS should not be a requirement. Perhaps we should remove scipy as well. They can be listed in the optional requirements

Requirements here should not be set to the latest version of things, they should be set to the oldest version that works with the package.

Reply:
I reverted these requirements. Should be good for now.